src/Common/src/TypeSystem/IL/Stubs/ArrayMethodILEmitter.cs

```diff
@@ -136,16 +135,19 @@ void EmitILForAccessor()
 
             codeStream.Emit(ILOpcode.ret);
 
-#if false
             codeStream.EmitLdc(0);
             codeStream.EmitLabel(rangeExceptionLabel1); // Assumes that there is one "int" pushed on the stack
             codeStream.Emit(ILOpcode.pop);
 
-            var tokIndexOutOfRangeCtorExcep = GetToken(GetException(kIndexOutOfRangeException).GetDefaultConstructor());
+            TypeDesc indexOutOfRangeException = _method.Context.GetWellKnownType(WellKnownType.IndexOutOfRangeException);
+            int tokIndexOutOfRangeCtorExcep = NewToken(indexOutOfRangeException.GetDefaultConstructor());
+#if false
             codeStream.EmitLabel(rangeExceptionLabel);
```

Comment:
Delete this label - looks like a unused left over (and rename the other one to just rangeExceptionLabel)?